submissions/Harnytskyi/html_css_popup/index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <link rel="stylesheet" href="css/style.css">  
```

Comment:
Metatag Viewport sets up the correct setting for the mobile view. 
https://serpstat.com/ru/blog/chto-takoe-metateg-viewport/
Although CSS styles for mobile is not required for this task, this meta tag is required for markup in general
`<meta name="viewport" content="width=device-width, initial-scale=1.0">`